roles/rclone/README.md

```diff
@@ -0,0 +1,3 @@
+Rclone is a command line program to manage files on cloud storage. It supports over 40 cloud storage products including S3.
```

Comment:
Can you make the following additions to this README?

- Add a note that explicitly says access keys are required to use this role
- Add a usage example